x-pack/docs/en/security/rbac.asciidoc

```diff
@@ -0,0 +1,38 @@
+:kib-privs: {kib} Privileges
+
+[role="xpack"]
+[[kibana-rbac]]
+=== Role Based Access Control (RBAC) in Kibana
+
+
+--
+
+NOTE: RBAC is an experimental opt-in feature. Additional functionality will be added in later versions of {kib}.
+
+--
+
+RBAC allows you to control access to {kib} objects, including Dashboards, Visualizations, Saved Searches, and Index Patterns.
+
+See <<security-rbac-settings,Configuring RBAC in {kib}>> for setup instructions.
+
+==== Features
+
+With RBAC enabled, you no longer need to grant user roles direct access to the {kib} index (e.g., `.kibana`).
+Instead, each user role can be assigned one or more {kib-privs}, which are used to grant access to the underlying {kib} objects.
+
+The default `kibana_user` and `kibana_dashboard_only_user` roles will not have any {kib-privs} assigned.
+When RBAC is first enabled, {kib} will create two new corresponding roles which should be used instead: `kibana_rbac_user` and `kibana_rbac_dashboard_only_user`.
+
+The `kibana` prefix is determined by your `xpack.security.rbac.application` setting, which defaults to `kibana`. If your `xpack.security.rbac.application` is set to `example`, then the corresponding roles would be named `example_rbac_user` and `example_rbac_dashboard_only_user`.
```

Comment:
Can we also mention here the note that you have below in the actual settings about this only being required/recommended to configure for multi-tenant Kibana?